cypress/cypress/e2e/EngageSphere_API.cy.js

```diff
@@ -0,0 +1,62 @@
+// Recupera clientes com sucesso (por exemplo, verifica o código de status 200)
+describe('Recuperação de Clientes', () => {
```

Comment:
Que tal algo como abaixo?

```suggestion
describe('GET /customers', () => {
```